openshift/template.yaml

```diff
@@ -181,6 +181,15 @@ objects:
                   value: ${PYPI_TRAINING_REPO}
                 - name: GREMLIN_QUERY_SIZE
                   value: "25"
+                - name: APP_SECRET_KEY
+                  valueFrom:
+                    secretKeyRef:
+                      name: jobs
+                      key: flask-app-secret-key
```

Comment:
We are Introducing a new Secret? Do we really need this..? 

If Yes, Have you put this in Vault..?

Reply:
No, it's is an existing secret which was used in Jobs API's and it goes by default with deployment scripts. I have added it here so its value can be set in requests for authentication in Jobs API,  I am not sure if in production its value is set to default, if yes then I feel values needs to be changed in vault as default value can be seen in deployment scripts on github.